src/task/period.rs

```diff
@@ -8,9 +8,23 @@ pub enum Period {
 }
 
 impl Period {
-    fn is_leap_year(year: i32) -> bool {
+    pub fn is_leap_year(year: i32) -> bool {
```

Comment:
Why have you made `is_leap_year` public? I think it’s not necessary.

Reply:
You're right, I changed it (and rebased on your changes)